lib/src/main/java/net/mossol/bot/connection/ToLocationInfoRequestConverter.java

```diff
@@ -0,0 +1,28 @@
+package net.mossol.bot.connection;
+
+import javax.annotation.Nullable;
+
+import net.mossol.bot.model.LocationInfo;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.linecorp.armeria.common.AggregatedHttpMessage;
+import com.linecorp.armeria.server.ServiceRequestContext;
+import com.linecorp.armeria.server.annotation.RequestConverterFunction;
+
+public class ToLocationInfoRequestConverter implements RequestConverterFunction {
```

Comment:
I understand the class name `ToLocationInfoRequestConverter` is originated that it will be used for `destination`, right? 

IMHO. 
But class's role seems simply converting data into LocationInfo.class. How about just naming `LocationInfoRequestConverter`?